populus/plugin.py

```diff
@@ -5,12 +5,23 @@
 )
 from populus.project import Project
 
+CACHE_KEY_MTIME = "populus/project/compiled_contracts_mtime"
+CACHE_KEY_CONTRACTS = "populus/project/compiled_contracts"
+
 
 @pytest.fixture(scope="session")
 def project(request):
     # This should probably be configurable using the `request` fixture but it's
     # unclear what needs to be configurable.
-    return Project()
+
+    # use pytest cache to preset the sessions project to recently compiled contracts
+    contracts = request.config.cache.get(CACHE_KEY_CONTRACTS, None)
+    mtime = request.config.cache.get(CACHE_KEY_MTIME, None)
+    project = Project(cached_contracts=contracts, contracts_mtime=mtime)
```

Comment:
Rather than complicating the constructor of the `Project` to work with this fixture what do you think about just manually setting these values on the `project` instance once instantiated?  I think I'm comfortable with that behavior as long as there is a basic test to cover that it works as expected.